frontend/drivers/platform_wiiu.c

```diff
@@ -559,8 +604,8 @@ static ssize_t wiiu_log_write(struct _reent *r, void *fd, const char *ptr, size_
 
    while(remaining > 0)
    {
-      int block = remaining < 1400 ? remaining : 1400;
-      ret = send(wiiu_log_socket, ptr, block, 0);
+      int block = remaining < 1472 ? remaining : 1472;
```

Comment:
why did you increase the size here ? 1500 MTU with 1472 data size is standard but not necessarily the case in every network.

Reply:
My thought was that since the wiiu doesn’t actually expose an MTU via any API, it’s best to use the de facto standard. Besides, few if any log messages are actually writing that much data in a single log message.

Reply:
still, why increase it ?

Reply:
The only reason for the change is to use a relatively standard packet size. 

But I think we can use the minimum size (don’t have the number in front of me, it’s 500-something bytes) and still have log messages fit in a single packet.